app/view_objects/assessor_interface/application_forms_index_view_object.rb

```diff
@@ -11,7 +11,7 @@ def application_forms
     ::Filters::State.apply(
       scope: application_forms_without_state_filter,
       params:
-    )
+    ).order(created_at: :desc)
```

Comment:
This is fine for now but thinking about it we're going to want submitted_at which we don't have. I'll card that up.